docs/site.md

```diff
@@ -39,7 +39,8 @@ lazy val unidocs = project
   .in(file("unidocs"))
   .enablePlugins(TypelevelUnidocPlugin) // also enables the ScalaUnidocPlugin
   .settings(
-    name := "woozle-docs"
+    name := "woozle-docs",
+    ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(core.jvm, heffalump)
```

Comment:
I'd still like to think about ways to automate this setting. It's tricky to do in general—you can't just ignore JS/native projects, because some modules may only be published for those.

Reply:
Btw @timwspence I think adding a setting like this should fix the cats-stm build.